src/types/spend_bundle.py

```diff
@@ -20,15 +20,16 @@ class SpendBundle(Streamable):
     """
 
     coin_solutions: List[CoinSolution]
-    aggregated_signature: G2Element
+    aggregated_signature: Optional[G2Element]
```

Comment:
Don't do this. Use the 0 signature instead (adding will not change the final signature).

Reply:
AKA `infinity`. See `debug_spend_bundle` below.

Reply:
I considered that, but I did this to save space when sending transactions. The serialization of the infinity element is also 96 bytes. What do you think?

Reply:
A `SpendBundle` without a signature will be a rare thing indeed.